6/stop-demo.sh

```diff
@@ -0,0 +1,2 @@
+docker ps | grep client-cluster | awk '{print $1}' | xargs -I {} docker stop {}
```

Comment:
This will also need to change. We should ideally run:

* silo cluster
* one client
* reasonably aggressive vegeta (500RPS at least)

Reply:
I left that comment ^^ thinking it was for the demo start script, but this script is correct, we will need to tear down the clients too.